src/building/suggested.md

```diff
@@ -58,7 +59,7 @@ Task] instead:
 }
 ```
 
-[Build Task]: https://code.visualstudio.com/docs/editor/tasks
+[build task]: https://code.visualstudio.com/docs/editor/tasks
```

Comment:
Did you mean to change this? I think this will break the link.

Reply:
oh no i didn't realize this was changed, mb